wiki/Game_Modifiers/en.md

```diff
@@ -1,306 +1,531 @@
+[Cinema]: /wiki/shared/mods/CM.png "Cinema mod icon"
+[1K]: /wiki/shared/mods/1K.png "1K mod icon"
+[2K]: /wiki/shared/mods/2K.png "2K mod icon"
+[3K]: /wiki/shared/mods/3K.png "3K mod icon"
+[4K]: /wiki/shared/mods/4K.png "4K mod icon"
+[5K]: /wiki/shared/mods/5K.png "5K mod icon"
+[6K]: /wiki/shared/mods/6K.png "6K mod icon"
+[7K]: /wiki/shared/mods/7K.png "7K mod icon"
+[8K]: /wiki/shared/mods/8K.png "8K mod icon"
+[9K]: /wiki/shared/mods/9K.png "9K mod icon"
+[Co-Op]: /wiki/shared/mods/COOP.png "Co-Op mod icon"
+[Random]: /wiki/shared/mods/RD.png "Random mod icon"
+[ScoreV2]: /wiki/shared/mods/SV2.png "ScoreV2 mod icon"
+[TP]: /wiki/shared/mods/TP.png "Target Practice (TP) mod icon"
+[10K]: /wiki/shared/mods/10K.png "10K mod icon"
+[FO]: /wiki/shared/mods/FO.png "Fade Out (FO) mod icon"
+[No Video]: /wiki/shared/mods/NV.png "No Video mod icon"
+
 # Game Modifiers
 
-A **game modifier** is one of several optional modifications to gameplay the player can select. Some modifiers (mods) lower difficulty in various ways but assign a score penalty by multiplying the final score with a modifier less than 1.00. Other mods increase difficulty to provide a greater challenge and also reward the player with a modifier greater than 1.00 to the final score. Players who use the Hidden and Flashlight mods also gain the possibility to attain Silver S and Silver SS grades. Finally, some mods can alter the gameplay experience without affecting difficulty (such as the Relax mod, which you just move your mouse to the hit objects).
+![Game mods screen][GM]
+
+A **game modifier** is one of several optional modifications to gameplay the player can select. Some modifiers (mods) lower difficulty in various ways but assign a score penalty by multiplying the final score with a modifier less than 1.00. Other mods increase difficulty to provide a greater challenge and also reward the player with a modifier greater than 1.00 to the final score. Players who use the Hidden and Flashlight mods will also gain the possibility to attain Silver S and Silver SS grades. Finally, some mods can alter the gameplay experience without affecting difficulty (such as the Relax mod, which the player only need to move the cursor to the hit objects).
 
 Game modifiers can be accessed by pressing `F1` at song select, or by clicking the `Mods` button near the lower left of the screen.
 
-Note that when multiple mods are applied, the product of the score multipliers is given instead of the sum of them. For example, when Hidden and Flashlight are applied at the same time, they give 1.1872x because (1.06 x 1.12) = 1.1872, rather than (1.06 + 1.12 - 1.00) = 1.18. *osu!* used to round this number down to the nearest hundredth (0.01), so Hidden and Flashlight together would appear to give a 1.18x multiplier, denying the supposed number round up. Thankfully, a patch corrected this, and now Hidden + Flashlight rounds correctly to 1.19x.
+![Song Selection's Toolbox][Toolbox]
```

Comment:
red arrows don't look great on images :/ ... maybe shrink down the cursor?
![screenshot167](https://user-images.githubusercontent.com/14264992/28501218-e9f86bd0-6f9c-11e7-8596-a3657d8c561e.jpg)